detection-rules/attachment_html_suspicious_comments.yml

```
@@ -0,0 +1,55 @@
+name: "Attachment: HTML With Suspicious Comments"
+description: "Detects HTML files under 100KB that contain duplicate or padding text in the form of literary quotes or common sayings within HTML comments."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments,
+          (
+            (
+              .file_type == "html"
+              or .file_extension in ("html", "xhtml", "mhtml")
+              or .content_type == "text/html"
+            )
+            and .size < 100000
+          )
+          and (
+            (
+              // targeting comments that pad the file with quotes from literature
+              // examples: "// Echoes of the past linger in silence.", "// The wind whispered secrets unknown.", "// Shadows tell stories in the dark."
+  
+              // count all HTML code comments that match our pattern
+              length(regex.extract(file.parse_text(.).text, '// [A-Z][ a-z ]+\.')) / 
```

i think regex.count does the same thing for this calculation.
```suggestion
              regex.count(file.parse_text(.).text, '// [A-Z][ a-z ]+\.') / 
```